qa-test-app/build.gradle

```diff
@@ -0,0 +1,126 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'com.mapbox.maps.token'
+apply from: "${rootDir}/gradle/script-git-version.gradle"
+apply from: "${rootDir}/gradle/ktlint.gradle"
+apply plugin: 'com.google.firebase.crashlytics'
+
+android {
+    compileSdkVersion androidVersions.compileSdkVersion
+
+    def getGitHash = { ->
+        try {
+            def stdout = new ByteArrayOutputStream()
+            exec {
+                commandLine 'git', 'rev-parse', '--short', 'HEAD'
+                standardOutput = stdout
+            }
+            return stdout.toString().trim()
+        } catch (Exception exception) {
+            return ""
+        }
+    }
+
+    defaultConfig {
+        applicationId "com.mapbox.navigation.qa_test_app"
+        minSdkVersion androidVersions.minSdkVersion
+        targetSdkVersion androidVersions.targetSdkVersion
+        multiDexEnabled true
+        versionCode gitVersionCode
+        versionName gitVersionName
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunnerArguments clearPackageData: 'true'
+        vectorDrawables.useSupportLibrary = true
+        buildConfigField "String", "GIT_HASH", "\"${getGitHash()}\""
+    }
+
+    testOptions {
+        execution 'ANDROIDX_TEST_ORCHESTRATOR'
+    }
+
+    signingConfigs {
+        release {
+            keyAlias System.getenv("CIRCLECI_ANDROID_KEYSTORE_ALIAS")
+            keyPassword System.getenv("CIRCLECI_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD")
+            storeFile file("$project.rootDir/navigation_testapp.keystore")
```

Comment:
please add this to `.gitignore`  and store all credentials to 1Password

Reply:
I am not sure if this is needed. We don't do it for other `example` modules in `2.x` as well as previous `example` module in `v1.x` as the credentials are already stored in `CircleCI`. However, I don't think this should be a blocker, as if this change is necessary then it is probably needed for all applicable modules.

cc @RingerJK @cafesilencio